packages/dids/tests/methods/did-key.spec.ts

```diff
@@ -1,13 +1,14 @@
-import type { Jwk } from '@web5/crypto';
-
-import { expect } from 'chai';
-import { LocalKeyManager } from '@web5/crypto';
-
 import type { DidDocument } from '../../src/types/did-core.js';
+import type { Jwk } from '@web5/crypto';
 import type { PortableDid } from '../../src/types/portable-did.js';
 
+import chaiAsPromised from 'chai-as-promised';
 import { DidErrorCode } from '../../src/did-error.js';
+import { LocalKeyManager } from '@web5/crypto';
 import { DidKey, DidKeyUtils } from '../../src/methods/did-key.js';
+import { expect, use } from 'chai';
+
+use(chaiAsPromised);
```

Comment:
While communicated poorly perhaps the plan is to remove `chaiAsPromised` from all of the `@web5` packages to clear the way for being able to migrate off Mocha and Chai at some time.

```suggestion
```